index.js

```diff
@@ -1,55 +1,55 @@
+  productsTypesTypeid: function productsTypeTypeid(typeId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/products/types/${typeId}`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  productsTypesTypeidLocations: function productsTypeTypeidLocations(typeId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/products/types/${typeId}/locations`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  productsLocationsLocationidTypes: function productsLocationsLocationidTypes(locationId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/products/locations/${locationId}/types`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  productsTypesTypeidLocationsLocationid: function productsTypesTypeidLocationsLocationid(typeId, locationId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/products/types/${typeId}/locations/${locationId}`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zones: function zones() {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesType: function zonesType(type) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/${type}`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesTypeZoneid: function zonesTypeZoneid(type, zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/${type}/${zoneId}`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesTypeZoneidForecast: function zonesTypeZoneidForecast(type, zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/${type}/${zoneId}/forecast`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesForecastZoneidObservations: function zonesForecastZoneidObservations(zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/forecast/${zoneId}/observations`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesForecastZoneidStations: function zonesForecastZoneidStations(zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/forecast/${zoneId}/stations`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
   helloWorld: function helloWorld() {console.log("Hello World")}
-};
```

Comment:
you're taking the piss

Reply:
i think my copy and paste omitted the newline